enterprise/core-edge/src/main/java/org/neo4j/coreedge/catchup/storecopy/edge/CopiedStoreRecovery.java

```diff
@@ -46,8 +47,32 @@ public CopiedStoreRecovery( Config config, Iterable<KernelExtensionFactory<?>> k
 
     public void recoverCopiedStore( File tempStore )
     {
-        GraphDatabaseService graphDatabaseService = newTempDatabase( tempStore );
-        graphDatabaseService.shutdown();
+        try
+        {
+            GraphDatabaseService graphDatabaseService = newTempDatabase( tempStore );
+            graphDatabaseService.shutdown();
+        }
+        catch ( Exception e )
+        {
+            if ( e.getCause().getCause() instanceof UpgradeNotAllowedByConfigurationException )
```

Comment:
@davidegrohmann  good idea. I'll add that as another commit. Want to merge this in now so @martinfurmanski  can do soak testing with the high limit record format